README.md

```diff
@@ -282,6 +282,27 @@ See also [related Jenkins documentation](https://www.jenkins.io/doc/book/pipelin
 
 It is also possible to let goveralls run the code coverage on its own without providing a coverage profile file.
 
+## TeamCity
+
+Store your Coveralls API token in `Environment Variables`:
+
+```
+COVERALLS_TOKEN=your_token_goes_here
+```
+
+Setup build steps:
+
+```
+$ go get github.com/mattn/goveralls
+$ export PULL_REQUEST_NUMBER=%teamcity.build.branch%
+$ goveralls -service teamcity -jobid %teamcity.build.id% -jobnumber %build.number%
+```
+
+`goveralls` automatically use the environment variable `COVERALLS_TOKEN` as the
+default value for `-repotoken`.
```

Comment:
What you mention here is valid for all CIs, however the documentation for other CIs do not mention how the environment variable is used in `goveralls`; I was actually thinking as part of #130 to start documenting all the CLI flags and environment variables used in the README.

Reply:
If we ever get to do that, this part should be removed because it would become redundant (or it should be mentioned in the documentation of the other CIs for symmetry).

Reply:
Yes, currently each CI section seems to be fair-play